### DISCUSSION TITLE
Dashboard not plotting in case of delayed trial processing.

### DISCUSSION THREAD

**Discussion by DeepankurK** [optuna/optuna-dashboard#406] [Q&A]
Hello Everyone , in my use case while using Optuna dashboard for objective function with study.optimize(), the dashboard is rendering perfectly fine but while using study.ask() and study.tell() to optimize my objective function, the dashboard just shows a blank white screen. I am not sure what the issue is and any work around or solution would be highly appreciated. 

Thanks!

**Comment by c-bata** [comment]
Sorry, the release of v0.9.0b7 has been delayed :bow:
https://github.com/optuna/optuna-dashboard/issues/401#issuecomment-1434678474

Or do you think it's another issue from #401?
